### PR TITLE
14th Small Improvements

### PR DESCRIPTION
**Pull request by zumbak04**
### Changelog:
- Reworked `Subdue Void Creature` of Shadow Priest. You no longer can summon N'raqi or C'thrax via `Subdue Void Creature` if your public and true religion isn't Shath'yar... BUT! You can summon Zoatroid!
- `Summon Demon` decision of Warlock no longer clear Demon-Minion modifier if you just want to summon Succubus.
- Contract with Demonic "Pets" from `Summon Demon` decision of Warlock now last only 10 years.
- Drek'thar is blind now.
- `Dwarf` trait has been renamed `Undersized`, characters with this trait can appear randomly.
- `Giant` trait has been renamed `Oversized`, characters with this trait can appear randomly.
- Added localization to existing ability of the grandmaster (leader) of the Cult of the Damned: the grandmaster can resurrect all types of undead regardless of grand-master's class level.
- Other characters now have bad opinion of Aggressive religions and cultures (ones that have free CBs) like Primitive, Vrykul religion group or Southern culture.
- AI rulers of Aggressive religions and cultures can't join or be invited in the Alliance or Horde.

### Secret сhangelog:
- Purged plural nouns in culture names. Culture names must be adjectives.
**_How to test:_** @DukeJeans or @MemeWizard , please, check changed files.

### CODE REVIEW

File: common/traits/00_traits.txt [hunk 1]
```diff
@@ -2025,3 +2019,5 @@ cruel = {
 	female_insult_adj = INSULT_CALLOUS
 	child_insult_adj = INSULT_WICKED
 }
+
+# Warcraft
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply common/traits/00_traits.txt]: Vanilla lines were deleted here.

File: common/traits/00_traits.txt [hunk 2]
```diff
@@ -928,32 +928,26 @@ ugly = {
...
 		giant
 	}
-
+	
 	martial = -1
-
+	
 	combat_rating = -15 #old value: -3
-
+
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply common/traits/00_traits.txt]: Copied lines from vanilla where spaces exist.

**Comment by zumbak04** [comment]
> if posible making it a one time use per summoning when you succeed on a mind control?

Doesn't that make the decision too expensive? 3 mana after all.

**Comment by zumbak04** [comment]
> Reminder for Zumbak: a higher mind control success with the zoatroid with giving it some kind of debuff (probably removing plot power or adding a -1 diplomacy?)

@Nayfis implemented!

File: events/wc_class_events.txt [hunk 3]
```diff
@@ -2717,16 +2739,86 @@ character_event = {
...
+	# Merciless one
+	option = {
+		trigger = {
+			NOT = { has_character_modifier = pet_merciless_one }
+		}
+		name = EVTOPTC_WCCLS_1213
+		
+		clr_character_flag = summon_void_creature_ongoing_flag
+		
+		add_character_modifier = { name = pet_merciless_one years = 10 }
```
Comment (MemeWizard) [inline events/wc_class_events.txt]: It's weird that you summon zoatroids for only 10 years, but demons don't "expire"

Comment (MemeWizard) [reply events/wc_class_events.txt]: Changed!

**Pull request merged**